apps/web/src/pages/SearchRequests/SearchPage/SearchPage.tsx

```diff
@@ -21,43 +36,10 @@ const SearchPage = () => {
         })}
       />
       <Hero
-        centered
-        title={intl.formatMessage({
-          defaultMessage: "Find digital talent",
-          id: "9Jkoms",
-          description: "Title displayed on hero for Search and Request pages.",
-        })}
-      >
-        <div
-          data-h2-background-color="base(foreground)"
-          data-h2-radius="base(rounded)"
-          data-h2-padding="base(x1) p-tablet(x2)"
-          data-h2-shadow="base(large)"
-          data-h2-text-align="base(center)"
-        >
-          <Heading
-            level="h2"
-            data-h2-margin="base(0, 0, x1, 0)"
-            data-h2-font-weight="base(400)"
-          >
-            {intl.formatMessage({
-              defaultMessage: "Are you looking to hire digital talent?",
-              id: "zMuDSL",
-              description:
-                "Heading on find talent page posing question to users.",
-            })}
-          </Heading>
-          <p>
-            {intl.formatMessage({
-              defaultMessage:
-                "If you are looking for talent, you have found the right place. Our talent database is open to most departments and agencies. Complete a request to find qualified candidates. Candidates are assessed for their skills and grouped into pools to meet your staffing needs.",
-              id: "F+LDbs",
-              description:
-                "Content displayed in the find talent page explaining the page and what it offers to users.",
-            })}
-          </p>
-        </div>
-      </Hero>
+        title={title}
+        subtitle="Discover talent using a set of comprehensive filters, including classification, languages, and skills."
```

Comment:
This needs to be wrapped in `intl.formatMessage`, otherwise it will not be able to be translated.

Reply:
Yep - I've already chatted with @esizer about new text and getting it properly implemented. There will be a separate list of instances in the requirements.

Reply:
Updated in [localized strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8485/commits/8978e0a26fbdfb8875ddaea4bf51bb96da639515)